### PR TITLE
Create electricity-expansion.yaml

### PR DESCRIPTION
**Pull request by sandrinecharousset**
added variables results of plan4EU capacity expansion

### CODE REVIEW

File: definitions/variable/technology/electricity-expansion.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# List of variables related to the evolution of the electricity system
+# results of capacity expansion models
+
+- New Capacity|Electricity|{Electricity Input}:
+    description: New capacity for Electricity Input
```
Comment (erikfilias) [inline definitions/variable/technology/electricity-expansion.yaml:5]:
```suggestion
    description: New capacity for {Electricity Input}
```

File: definitions/variable/technology/electricity-expansion.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      this capacity is added to the existing capacity
+    unit: MWh
+- New Storage|Electricity|Enegy Storage System|{Energy Storage System Type}:
+    description: New storage capacity for Electricity Input
```
Comment (erikfilias) [inline definitions/variable/technology/electricity-expansion.yaml:9]:
```suggestion
    description: New storage capacity for {Enery Storage System Type},
```

File: definitions/variable/technology/electricity-expansion.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+      this capacity is added to the existing capacity
+    unit: MWh
+- New Capacity|Network|Electricity:
+    description: New capacity for Electricity Input
```
Comment (erikfilias) [inline definitions/variable/technology/electricity-expansion.yaml:13]:
```suggestion
    description: New capacity for the electricity network, 
```

File: definitions/variable/technology/electricity-expansion.yaml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+      this capacity is added to the existing capacity
+    unit: MWh
+- Removed Capacity|Electricity|{Electricity Input}:
+    description: Removed capacity for Electricity Input
```
Comment (erikfilias) [inline definitions/variable/technology/electricity-expansion.yaml:17]:
```suggestion
    description: Removed capacity for {Electricity Input},
```

File: definitions/variable/technology/electricity-expansion.yaml [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+      this capacity is substracted from the existing capacity
+    unit: MWh
+- Removed Storage|Electricity|Enegy Storage System|{Energy Storage System Type}:
+    description: Removed storage capacity for Electricity Input
```
Comment (erikfilias) [inline definitions/variable/technology/electricity-expansion.yaml:21]:
```suggestion
    description: Removed storage capacity for {Energy Storage System Type},
```

File: definitions/variable/technology/electricity-expansion.yaml [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+      this capacity is substracted from the existing capacity
+    unit: MWh
+- Removed Capacity|Network|Electricity:
+    description: Removed capacity for Electricity Input
```
Comment (erikfilias) [inline definitions/variable/technology/electricity-expansion.yaml:25]:
```suggestion
    description: Removed capacity for the electricity network,
```

File: definitions/variable/technology/electricity-expansion.yaml [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+      this capacity is added to the existing capacity
+    unit: MWh
+- New Storage|Electricity|Enegy Storage System|{Energy Storage System Type}:
+    description: New storage capacity for Electricity Input
+      this capacity is added to the existing capacity
+    unit: MWh
+- New Capacity|Network|Electricity:
+    description: New capacity for Electricity Input
+      this capacity is added to the existing capacity
+    unit: MWh
```
Comment (danielhuppmann) [inline definitions/variable/technology/electricity-expansion.yaml:15]: Why is network capacity measured in energy (MWh), not in power (MW)?

File: definitions/variable/technology/electricity-expansion.yaml [hunk 8]
```diff
@@ -0,0 +1,27 @@
+# List of variables related to the evolution of the electricity system
+# results of capacity expansion models
+
+- New Capacity|Electricity|{Electricity Input}:
```
Comment (danielhuppmann) [inline definitions/variable/technology/electricity-expansion.yaml:4]: I would rather name this variable as "Capacity Addition".

File: definitions/variable/technology/electricity-expansion.yaml [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+    description: New capacity for Electricity Input
+      this capacity is added to the existing capacity
+    unit: MWh
+- New Storage|Electricity|Enegy Storage System|{Energy Storage System Type}:
```
Comment (danielhuppmann) [inline definitions/variable/technology/electricity-expansion.yaml:8]: This should be "Capacity Addition|Storage|..."

File: definitions/variable/technology/electricity-expansion.yaml [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+    description: New storage capacity for Electricity Input
+      this capacity is added to the existing capacity
+    unit: MWh
+- New Capacity|Network|Electricity:
+    description: New capacity for Electricity Input
+      this capacity is added to the existing capacity
+    unit: MWh
+- Removed Capacity|Electricity|{Electricity Input}:
```
Comment (danielhuppmann) [inline definitions/variable/technology/electricity-expansion.yaml:16]: I suggest to rename this to "Capacity Decommissioning|..."

**Review by danielhuppmann — COMMENTED** [review]
See suggestions inline

File: definitions/variable/technology/electricity-expansion.yaml (hunk 8 above)
Comment (sandrinecharousset) [reply definitions/variable/technology/electricity-expansion.yaml:4]: good idea! will do the change

File: definitions/variable/technology/electricity-expansion.yaml (hunk 10 above)
Comment (sandrinecharousset) [reply definitions/variable/technology/electricity-expansion.yaml:16]: good suggestion

File: definitions/variable/technology/electricity-expansion.yaml (hunk 7 above)
Comment (sandrinecharousset) [reply definitions/variable/technology/electricity-expansion.yaml:15]: and same thing for Electricity|{Electricity Input}

File: definitions/variable/technology/electricity-expansion.yaml [hunk 11]
```diff
@@ -1,27 +1,27 @@
 # List of variables related to the evolution of the electricity system
 # results of capacity expansion models
 
-- New Capacity|Electricity|{Electricity Input}:
+- Capacity Addition|Electricity|{Electricity Input}:
```
Comment (erikfilias) [inline definitions/variable/technology/electricity-expansion.yaml:4]:
```suggestion
- Commissioned Capacity|Electricity|{Electricity Input}:
```

File: definitions/variable/technology/electricity-expansion.yaml [hunk 12]
```diff
@@ -1,27 +1,27 @@
...
     description: New capacity for {Electricity Input}
       this capacity is added to the existing capacity
-    unit: MWh
-- New Storage|Electricity|Enegy Storage System|{Energy Storage System Type}:
+    unit: MW
+- Storage Addition|Electricity|Enegy Storage System|{Energy Storage System Type}:
```
Comment (erikfilias) [inline definitions/variable/technology/electricity-expansion.yaml:8]:
```suggestion
- Commissioned Storage|Electricity|Enegy Storage System|{Energy Storage System Type}:
```

File: definitions/variable/technology/electricity-expansion.yaml [hunk 13]
```diff
@@ -1,27 +1,27 @@
...
     description: New storage capacity for {Enery Storage System Type},
       this capacity is added to the existing capacity
     unit: MWh
-- New Capacity|Network|Electricity:
+- Capacity Addition|Network|Electricity:
```
Comment (erikfilias) [inline definitions/variable/technology/electricity-expansion.yaml:12]:
```suggestion
- Commissioned Capacity|Network|Electricity:
```

File: definitions/variable/technology/electricity-expansion.yaml [hunk 14]
```diff
@@ -1,27 +1,27 @@
...
     description: New capacity for the electricity network, 
       this capacity is added to the existing capacity
-    unit: MWh
-- Removed Capacity|Electricity|{Electricity Input}:
+    unit: MW
+- Capacity Decommissioning|Electricity|{Electricity Input}:
```
Comment (erikfilias) [inline definitions/variable/technology/electricity-expansion.yaml:16]:
```suggestion
- Decommissioned Capacity|Electricity|{Electricity Input}:
```

File: definitions/variable/technology/electricity-expansion.yaml [hunk 15]
```diff
@@ -1,27 +1,27 @@
...
     description: Removed capacity for {Electricity Input},
       this capacity is substracted from the existing capacity
-    unit: MWh
-- Removed Storage|Electricity|Enegy Storage System|{Energy Storage System Type}:
+    unit: MW
+- Storage Decommissioning|Electricity|Enegy Storage System|{Energy Storage System Type}:
```
Comment (erikfilias) [inline definitions/variable/technology/electricity-expansion.yaml:20]:
```suggestion
- Decommissioned Storage|Electricity|Enegy Storage System|{Energy Storage System Type}:
```

File: definitions/variable/technology/electricity-expansion.yaml [hunk 16]
```diff
@@ -1,27 +1,27 @@
...
     description: Removed storage capacity for {Energy Storage System Type},
       this capacity is substracted from the existing capacity
     unit: MWh
-- Removed Capacity|Network|Electricity:
+- Capacity Decommissioning|Network|Electricity:
```
Comment (erikfilias) [inline definitions/variable/technology/electricity-expansion.yaml:24]:
```suggestion
- Decommissioned Capacity|Network|Electricity:
```

**Review by erikfilias — COMMENTED** [review]
I would use "Capacity Invested" and "Capacity Retired".

**Review by erikfilias — APPROVED** [review]

**Pull request merged**